src/posthog-surveys-types.ts

```diff
@@ -110,4 +110,5 @@ export interface Survey {
     } | null
     start_date: string | null
     end_date: string | null
+    events: string[]
```

Comment:
to prevent breaking changes headaches (they are very hard to do in posthog-js once we release to users) - consider fleshing out the data model here, since we already know this wouldn't be sufficient.

Reply:
posthog-js, even with a major version update, updates immediately for everyone using the snippet as soon as we update it on our main posthog repo, which means things can break for users unknowingly, so posthog-js needs to always be backwards compatible